app/Platform/Controllers/GameDevInterestController.php

```diff
@@ -40,7 +40,7 @@ public function __invoke(Request $request): View
             ->orderBy('UserAccounts.User')
             ->pluck('UserAccounts.User');
 
-        return view('platform.components.game.dev-interest-page', [
+        return view('components.game.dev-interest-page', [
```

Comment:
nice, thanks - I forgot to check if there are any other `*-page.blade.php` left.

ideally the route parameter should match a model name where the route key, usually the db ID, will cause the model to be loaded so it can be bound automatically and make it available to the view/controller.

moved to `pages.game.[game].dev-interest` in [latest](https://github.com/RetroAchievements/RAWeb/pull/2229/commits/59f3821c0025c9dc1f140736a3562ceec5e97e08).